tests/x11regressions/tomboy/tomboy_StartNoteCannotBeDeleted.pm

```diff
@@ -21,32 +21,19 @@ sub run {
 
     # select "start note", to see that start note cann't be deleted
     send_key "tab";
-    sleep 2;
     send_key "down";
-    sleep 2;
     send_key "down";
-    sleep 2;
     send_key "ret";
-    sleep 2;
-    wait_idle;
-    check_screen "tomboy_delete_0", 5;    # to see if the delete buttom is avaiable
-    sleep 2;
+    check_screen "tomboy_delete_0";    # to see if the delete buttom is avaiable
```

Comment:
s/avaiable/available
Not sure if we need check_screen if we don't check the results.